### ISSUE TITLE
BakedLightmap data is not assigned wen saved as a file.

### ISSUE THREAD

**Issue by Janders1800** [godotengine/godot#31646]
**Godot version:**
3.1.1 Stable

**OS/device including version:**
Zorin OS Core 12 (Ubuntu 16.04 LTS)

**Issue description:**
If you save the BakedLightmap data as a file and then save the scene, the route to the data file gets lost and returns to save the data inside the scene file.

**Steps to reproduce:**
- Bake a lightmap.
- Save the lightmap data as a file.
- Save the scene.

**Minimal reproduction project:**
[LightMaps.zip](https://github.com/godotengine/godot/files/3537957/LightMaps.zip)

**Comment by Quincy9000** [comment]
Yup, same here. Still happens to me on 3.2.2.stable.mono as well.

**Comment by Quincy9000** [comment]
I have found a sort of fix. 

1.

**Comment by Janders1800** [comment]
This is no longer a problem since long ago. Closed.
